src/app/modules/form/admin-form/__tests__/admin-form.assistance.controller.spec.ts

```diff
@@ -80,7 +91,7 @@ describe('admin-form.assistance.controller', () => {
       })
 
       // Mock OpenAI API throwing an error
-      MockedOpenAIClient.prototype.chat.completions.create = jest
+      MockedOpenAIClient.prototype.getChatCompletions = jest
         .fn()
         .mockRejectedValue(new Error('Some random error message'))
 
```

Comment:
since this is a controller test, it suffices to mock the behavior of the service, e.g.

```
      const generateQuestionsSpy = jest
        .spyOn(AdminFormAssistanceService, 'generateQuestions')
        .mockReturnValueOnce(errAsync(new AssistanceConnectionError()) as any)

```

Reply:
thank you!!